task_15/test/pageobjects/secure.page.js

```diff
@@ -0,0 +1,17 @@
+
+
+const Page = require('./page');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/jkalandarov/epam_stage_2/pullRequest?prid=8887246)
```suggestion
const Page = require("./page");
```